_drafts/oryx/technology.md

```diff
@@ -0,0 +1,66 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies.
+When possible and appropriate, Oryx adheres to standards for both technologies and tools.
+Oryx strives to minimize the use of build tools and instead utilize the native capabilities of the web, as much as possible.
+Oryx aims to use JavaScript during development, but tries avoid the need to run JavaScript in the browser. Wherever possible, HTML and CSS are preferred to render a fast loading experience cross different screen sizes.
+
+We will examine both the deployed technology stack and the technologies utilized for development in order to gain a more comprehensive understanding of the technologies.
+
+## Deployed stack:
+
+### HTML & CSS & JS
+HTML, CSS, and JS are considered to be the fundamental and essential set of web technologies used for creating web pages. HTML is responsible for the structure and content of the page, CSS is used for styling and layout, and JS adds interactivity and dynamic behavior to the page. Together, these three technologies form the foundation of modern web development.
+
+In Oryx we prioritize CSS as the primary means of styling and layout rendering, embracing a CSS-first approach. We favor CSS over JavaScript for its inherent capability to handle styling and layout tasks without the need for additional JavaScript logic.
+
+We are  to avoid loading JS in the client unless there's a need; we gradually load snippets of JS rather than a full JS framework.
+
+In the Oryx framework, our approach is to minimize the loading of JavaScript on the client side unless it is necessary. Instead of loading a full JavaScript framework, we opt for gradually loading snippets of JavaScript as required.
+
+### Web Components
+Web Components are a set of web platform APIs that allow developers to create custom, reusable, and encapsulated HTML elements.
+
+### Web hosting
+Any hosting provider can be used, but SSR specific build will require additional configuration.
```

Comment:
We can add some meet to this section: 

Hosting an oryx application would benefit from a couple of capabilities:
- build process and integration into the pipeline
- configuration of http headers (security, caching, etc.)
- CDN integration
- configuration of rewrites and redirects
- integration with lambda functions